### PR TITLE
Update dead link (Fix #96)

### LINKED ISSUE #96: Dead link on the web
I found the link problem in **Git issue tutorial**.

https://treehouses.io/#!./pages/vi/github-issues.md
> NOTE: Remember to always sync your fork before starting to work on a new issue. To sync your fork you can follow the process in the **previous step**.

This block contains a dead link. I known that you refer to this link [https://treehouses.io/#!./pages/vi/gitrepositories.md](https://treehouses.io/#!./pages/vi/gitrepositories.md) which has the **Sync Your Fork** section.

### PR DESCRIPTION
**Pull request by phamduchongan93**
Fix the broken link:

> NOTE: Remember to always sync your fork before starting to work on a new issue. To sync your fork you can follow the process in the previous **step**.

[raw githack link](https://raw.githack.com/phamduchongan93/phamduchongan93.github.io/fix_issue_96/#!pages/vi/github-issues.md)
This PR close #96

### CODE REVIEW

File: pages/vi/profiles/xdmtk.md [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
-```
+
```
Comment (dogi) [inline pages/vi/profiles/xdmtk.md]: this file is too much

**Review by dogi — CHANGES_REQUESTED** [review]
@phamduchongan93 please use `git rm ...` to remove the additional file

**Review by hiroTochigi — COMMENTED** [review]
image
It does not render photo correctly.

**Comment by lmmrssa** [comment]
@phamduchongan93 you should keep master branch synchronized and always create branch from master not from previous branch

**Comment by phamduchongan93** [comment]
Yeah, I understood the issue, but I'm struggling to keep the integrity of my master branch. There is some error during the forking process that I'm trying to figure out.

**Review by dogi — APPROVED** [review]

**Review by lillyxxx — APPROVED** [review]

**Pull request merged**